src/web/metrics.rs

```diff
@@ -72,29 +78,64 @@ lazy_static::lazy_static! {
     )
     .unwrap();
 
-    pub static ref CONCURRENT_DB_CONNECTIONS: IntGauge = register_int_gauge!(
-        "docsrs_db_connections",
-        "The number of currently used database connections"
+    pub static ref USED_DB_CONNECTIONS: IntGauge = register_int_gauge!(
+        "docsrs_used_db_connections",
```

Comment:
Let's not rename metrics if possible.

Reply:
It’s renamed because it does something differently now, only reporting the number of currently used connections instead of the total number